src/plugins/collision/collider/mod.rs

```diff
@@ -92,209 +91,457 @@ pub trait ScalableCollider: AnyCollider {
+#[cfg_attr(feature = "serialize", reflect(Serialize, Deserialize))]
+#[cfg_attr(all(feature = "3d", feature = "collider-from-mesh"), derive(Default))]
+#[cfg_attr(all(feature = "3d", feature = "collider-from-mesh"), reflect(Default))]
+#[allow(missing_docs)]
+pub enum ColliderConstructor {
+    /// Constructs a collider with [`Collider::circle`].
+    #[cfg(feature = "2d")]
+    Circle { radius: Scalar },
+    /// Constructs a collider with [`Collider::sphere`].
+    #[cfg(feature = "3d")]
+    Sphere { radius: Scalar },
+    /// Constructs a collider with [`Collider::ellipse`].
+    #[cfg(feature = "2d")]
+    Ellipse {
+        half_width: Scalar,
+        half_height: Scalar,
+    },
+    /// Constructs a collider with [`Collider::rectangle`].
+    #[cfg(feature = "2d")]
+    Rectangle { x_length: Scalar, y_length: Scalar },
+    /// Constructs a collider with [`Collider::cuboid`].
+    #[cfg(feature = "3d")]
+    Cuboid {
+        x_length: Scalar,
+        y_length: Scalar,
+        z_length: Scalar,
+    },
+    /// Constructs a collider with [`Collider::round_rectangle`].
+    #[cfg(feature = "2d")]
+    RoundRectangle {
+        x_length: Scalar,
+        y_length: Scalar,
+        border_radius: Scalar,
+    },
+    /// Constructs a collider with [`Collider::round_cuboid`].
+    #[cfg(feature = "3d")]
+    RoundCuboid {
+        x_length: Scalar,
+        y_length: Scalar,
+        z_length: Scalar,
+        border_radius: Scalar,
+    },
+    /// Constructs a collider with [`Collider::cylinder`].
+    #[cfg(feature = "3d")]
+    Cylinder { height: Scalar, radius: Scalar },
+    /// Constructs a collider with [`Collider::cone`].
+    #[cfg(feature = "3d")]
+    Cone { height: Scalar, radius: Scalar },
+    /// Constructs a collider with [`Collider::capsule`].
+    Capsule { height: Scalar, radius: Scalar },
+    /// Constructs a collider with [`Collider::capsule_endpoints`].
+    CapsuleEndpoints {
+        a: Vector,
+        b: Vector,
+        radius: Scalar,
+    },
+    /// Constructs a collider with [`Collider::halfspace`].
+    Halfspace { outward_normal: Vector },
+    /// Constructs a collider with [`Collider::segment`].
+    Segment { a: Vector, b: Vector },
+    /// Constructs a collider with [`Collider::triangle`].
+    Triangle { a: Vector, b: Vector, c: Vector },
+    /// Constructs a collider with [`Collider::regular_polygon`].
+    #[cfg(feature = "2d")]
+    RegularPolygon { circumradius: f32, sides: usize },
+    /// Constructs a collider with [`Collider::polyline`].
+    Polyline {
+        vertices: Vec<Vector>,
+        indices: Option<Vec<[u32; 2]>>,
+    },
+    /// Constructs a collider with [`Collider::trimesh`].
+    Trimesh {
+        vertices: Vec<Vector>,
+        indices: Vec<[u32; 3]>,
+    },
+    /// Constructs a collider with [`Collider::trimesh_with_config`].
+    TrimeshWithConfig {
+        vertices: Vec<Vector>,
+        indices: Vec<[u32; 3]>,
+        flags: TriMeshFlags,
+    },
```

Comment:
These should be `TriMesh`

Reply:
Changed all to `Trimesh` instead per discussion on Discord [`526135e` (#378)](https://github.com/Jondolf/bevy_xpbd/pull/378/commits/526135e1fa5d37b1a1616b7a383574b5768f8284)